client/my-sites/domains/domain-search/index.tsx

```diff
@@ -173,7 +204,7 @@ class DomainSearch extends Component {
 
 		let registration = domainRegistration( {
 			domain,
-			productSlug,
+			productSlug: productSlug as string,
```

Comment:
I dislike this immensely, but drew a blank a better approach than just heavy-handedly sidestepping the potential `null` value.

Reply:
Yep :(